mediatranslation/generated/com/google/cloud/mediatranslation/v1beta1/speechtranslationservicesettings/streamingtranslatespeech/SyncStreamingTranslateSpeech.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Google LLC
```

Comment:
All files under generated should not be migrated

Reply:
Oh good point. They have been taken out.